ground/src/main/java/com/google/android/ground/ui/common/BaseMapViewModel.kt

```diff
@@ -236,17 +237,17 @@ constructor(
   }
 
   /** Requests moving the map camera to [coordinates]. */
-  private fun panCamera(coordinates: Coordinates) {
-    updateMapCamera(CameraPosition(coordinates))
+  private fun panCamera(coordinates: Coordinates, shouldAnimate: Boolean) {
```

Comment:
"pan" implies animation, and the boolean arg makes these calls a little harder to grok. Perhaps it would be clearer if we had `panCamera` and `panAndZoomCamera` which animate, and `setCameraPosition` which does not?

Reply:
Thanks for the suggestion!